server/queue/queue-api/src/main/java/org/apache/james/queue/api/MailQueueFactory.java

```diff
@@ -38,9 +63,17 @@
      * @param name
      * @return queue
      */
-    Optional<T> getQueue(MailQueueName name);
+    default Optional<T> getQueue(MailQueueName name) {
+        return getQueue(name, defaultPrefetchCount());
+    }
+
+    Optional<T> getQueue(MailQueueName name, PrefetchCount count);

```

Comment:
Can we get javadoc on what prefetch count is?

Reply:
Yes